src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) != null) {
+            if (password.equals(userService.findByEmail(email).getPassword())) return true;
+        }
```

Comment:
I'd recommend calling findByEmail() method only once. Also, these ifs could be easily simplified and the solution will contain only one line (consider using '&&' operator)